gurdy:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 50
  
  # Position Controllers ---------------------------------------
  # Upper leg controllers
  head_upperlegM1_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: head_upperlegM1_joint
    pid: {p: 5.0, i: 1.0, d: 0.5}

  head_upperlegM2_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: head_upperlegM2_joint
    pid: {p: 5.0, i: 1.0, d: 0.5}

  head_upperlegM3_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: head_upperlegM3_joint
    pid: {p: 5.0, i: 1.0, d: 0.5}

  head_upperlegM4_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: head_upperlegM4_joint
    pid: {p: 5.0, i: 1.0, d: 0.5}

  head_upperlegM5_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: head_upperlegM5_joint
    pid: {p: 5.0, i: 1.0, d: 0.5}

  head_upperlegM6_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: head_upperlegM6_joint
    pid: {p: 5.0, i: 1.0, d: 0.5}

  # Lower leg controllers
  upperlegM1_lowerlegM1_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: upperlegM1_lowerlegM1_joint
    pid: {p: 3.0, i: 1.0, d: 0.5}

  upperlegM2_lowerlegM2_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: upperlegM2_lowerlegM2_joint
    pid: {p: 3.0, i: 1.0, d: 0.5}

  upperlegM3_lowerlegM3_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: upperlegM3_lowerlegM3_joint
    pid: {p: 3.0, i: 1.0, d: 0.5}

  upperlegM4_lowerlegM4_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: upperlegM4_lowerlegM4_joint
    pid: {p: 3.0, i: 1.0, d: 0.5}

  upperlegM5_lowerlegM5_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: upperlegM5_lowerlegM5_joint
    pid: {p: 3.0, i: 1.0, d: 0.5}

  upperlegM6_lowerlegM6_joint_position_controller:
    type: effort_controllers/JointPositionController
    joint: upperlegM6_lowerlegM6_joint
    pid: {p: 3.0, i: 1.0, d: 0.5}